packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,8 +3,11 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-panel-action-background-color: defines the background color of an action sub-component inside the component.
+ * @prop --calcite-panel-action-indicator-color: defines the indicator color of an action sub-component inside the component.
```

Comment:
AFAIK Indicator can't be set on these internally rendered Action cc @driskull to confirm. A user would have control over any slotted Actions to set this directly. Applies to other components.

Reply:
Yes, that's correct. The indicator isn't used on these internally rendered actions.